### PR TITLE
Worked on must/should/may wording

### PR DESCRIPTION
**Pull request by arnoldblinn**

### CODE REVIEW

File: Domain Connect Spec Draft.adoc [hunk 1]
```diff
@@ -1978,18 +2017,18 @@ r2.example.com. TXT v=spf1 include:_spf.newsletter.net -all
...
 
-If the existing TXT record makes the merging operation not possible, the DNS provider shall handle this situation the same way as a conflict and either let the end-user resolve it in the UX (both in Synchronous and Asynchronous flow) or return the conflict as an error in the Asynchronous flow unless the _force=true_ parameter is used, effectively removing the existing record.
+If the existing TXT record makes the merging operation not possible, the DNS provider must handle this situation the same way as a conflict and either let the end-user resolve it in the UX (both in Synchronous and Asynchronous flow) or return the conflict as an error in the Asynchronous flow unless the _force=true_ parameter is used, effectively removing the existing record.
 
-Service providers shall avoid checking content of TXT SPF record, as it might be strongly influenced by the DNS Provider merging strategy and user actions.
+Service providers should avoid checking content of TXT SPF record, as it might be strongly influenced by the DNS Provider merging strategy and user actions.
```
Comment (pawel-kow) [inline Domain Connect Spec Draft.adoc]: shall avoid exact match checking

File: Domain Connect Spec Draft.adoc [hunk 2]
```diff
@@ -172,68 +173,68 @@ To attach a domain name to a service provided by a Service Provider, the
...
 
-The user would first enter their domain name at the Service Provider
+The user first enters their domain name at the Service Provider
 website.
 
 image::.//media/image1.png[image,width=500,height=325]
 
 After the Service Provider determines the DNS Provider using discovery, 
-the Service Provider would typically display a link to the user indicating 
+the Service Provider should display a link to the user indicating
```
[the remark this answers is not in the document]
Comment (arnoldblinn) [reply Domain Connect Spec Draft.adoc]: High level description; won't debate should vs. may on this one.  Will take the recommendation.

File: Domain Connect Spec Draft.adoc [hunk 3]
```diff
@@ -172,68 +173,68 @@ To attach a domain name to a service provided by a Service Provider, the
...
 if necessary.
 
 image::.//media/image3.png[image,width=495,height=318]
 
-After authenticating at the DNS Provider, the DNS Provider would first verify
-the domain name is owned by the user. The DNS Provider would also verify
-other parameters passed in are valid, and would prompt the user for consent to 
-make the changes to DNS. The DNS Provider could also warn
+After authenticating at the DNS Provider, the DNS Provider must verify
+the domain name is owned by the user. The DNS Provider should also verify
```
[the remark this answers is not in the document]
Comment (arnoldblinn) [reply Domain Connect Spec Draft.adoc]: Should vs. Shall?  Shall is future, should tense is consistent with rest.  Will leave this one be...

File: Domain Connect Spec Draft.adoc [hunk 4]
```diff
@@ -245,50 +246,50 @@ over time.
...
+Instead of applying the DNS changes on user consent, OAuth access is
+granted to the Service Provider. An OAuth access code is generated and
 handed back to the Service Provider. The Service Provider then requests
 an access (bearer) token.
 
-The permission granted in the oAuth token is a right for the Service
+The permission granted in the OAuth token is a right for the Service
 Provider to apply a requested template (or templates) to the specific
-domain (and specific subdomains) owned by a specific user.
+domain (and specific subdomains) owned by a specific user at the DNS Provider.
```
[the remark this answers is not in the document]
Comment (arnoldblinn) [reply Domain Connect Spec Draft.adoc]: Done

File: Domain Connect Spec Draft.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (arnoldblinn) [reply Domain Connect Spec Draft.adoc]: Like this technicality fix.

[the remark this answers is not in the document]
Comment (pawel-kow) [reply Domain Connect Spec Draft.adoc]: @arnoldblinn sorry for confusion. I'm typically using SHALL, which in notation of rfc2119 is equal to MUST. I guess verification of parameters (like signatures etc.) is obligatory

File: Domain Connect Spec Draft.adoc [hunk 5]
```diff
@@ -500,40 +504,40 @@ GET
...
 ===== New Browser Window
 
 The first is through a new browser tab or in a popup browser window. 
-The DNS Provider would sign the user
-in if necessary, verify domain ownership, and ask for confirmation
+The DNS Provider signs the user
+in if necessary, verifies domain ownership, and asks for confirmation
 before application of the template. After application of the template,
-the DNS Provider would automatically close the browser tab or window.
+the DNS Provider should automatically close the browser tab or window.
```
[the remark this answers is not in the document]
Comment (arnoldblinn) [reply Domain Connect Spec Draft.adoc]: Not sure about this one....provider could have a button to do it.

File: Domain Connect Spec Draft.adoc [hunk 6]
```diff
@@ -172,68 +173,68 @@ To attach a domain name to a service provided by a Service Provider, the
...
 image::.//media/image4.png[image,width=489,height=312]
 
-Assuming the user grants this consent, the DNS changes would be applied.
+Assuming the user grants this consent, the DNS changes are be applied.
 
-Upon successful application of the DNS changes, if invoked in a pop-up
-window or tab the browser window would be closed. If invoked in place the user
-would be redirected back to the Service Provider.
+If invoked in a pop-up window or tab, the browser window should be closed
+after the changes are applied. If invoked in place, the user must be navigated back
```
[the remark this answers is not in the document]
Comment (arnoldblinn) [reply Domain Connect Spec Draft.adoc]: Not sure.  Some providers have a button and make the user do it.

File: Domain Connect Spec Draft.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (arnoldblinn) [reply Domain Connect Spec Draft.adoc]: Actually, you are right.

File: Domain Connect Spec Draft.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (arnoldblinn) [reply Domain Connect Spec Draft.adoc]: Actually, you are right from a spec POV.

**Comment by arnoldblinn** [comment]
I think I caught all of them.

**Pull request merged**